### PR TITLE
Added XML Comments

### PR DESCRIPTION
**Pull request by Chandler-Davidson**
Added XML comments to public classes, functions, and properties following [Issue #46](https://github.com/tdwright/contabs/issues/46). Let me know if any improvements need to be made.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -27,17 +27,16 @@ for:
   
   build:
     publish_nuget: true
-
```
Comment (tdwright) [inline appveyor.yml]: Wasn't expecting to see any changes to this file...?

File: ConTabs/Column.cs [hunk 2]
```diff
@@ -7,14 +7,45 @@
...
+		/// The defined type of the given property
+		/// </summary>
         public Type SourceType { get; set; }
+
+		/// <summary>
+		/// The name of the given property
+		/// </summary>
         public string PropertyName { get; private set; }
+
+		/// <summary>
```
Comment (tdwright) [inline ConTabs/Column.cs]: Worth mentioning this can be overwritten as desired? And is used for the column heading.

File: ConTabs/Column.cs [hunk 3]
```diff
@@ -7,14 +7,45 @@
...
+		/// </summary>
         public string FormatString { get; set; }
+
+		/// <summary>
+		/// A control to show/hide the column
+		/// </summary>
         public bool Hide { get; set; }
+
+		/// <summary>
+		/// The method the column displays long strings
```
Comment (tdwright) [inline ConTabs/Column.cs]: "The method the column _uses to_ display long strings"?

File: ConTabs/ConTabs.csproj [hunk 4]
```diff
@@ -18,4 +18,7 @@
     <DebugType>full</DebugType>
     <DebugSymbols>True</DebugSymbols>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard1.3|AnyCPU'">
```
Comment (tdwright) [inline ConTabs/ConTabs.csproj]: Is this needed? Does it make ConTabs incompatible with projects targetting .NET Standard versions less than 1.3? I think we currently target 1.2...

**Comment by tdwright-itg** [comment]
Hi @Chandler-Davidson - did you mean to close this? Hope I didn't put you off!

**Comment by Chandler-Davidson** [comment]
I messed some stuff up, so I'm making a new pull request. I appreciate the
patience!

**Comment by tdwright-itg** [comment]
Hey no worries. I look forward to the next one! 😃

**Pull request closed without merging**